### DISCUSSION TITLE
Command add armor stand

### DISCUSSION THREAD

**Discussion by arteffAA** [Zrips/CMI#6909] [Feature Requests]
Hello, you have a very cool and flexible rack editor, but there is one drawback: you cannot set a command to the armor stand (
ip click on the rack - the command is executed)

**Comment by AVuTuVA** [comment]
I also think that this function is needed in the plugin.

**Comment by yonakj** [comment]
this feature will be very useful

**Comment by dan28000** [comment]
Well you can use interactive commands for that or just some other simple plugin
But still can be useful

**Comment by Zrips** [comment]
/cmi ic command can in general achieve that already. Having new system to attach command to specific armorstands would be quite deprecated at this point, unless you want to make some spannable armor stands with preset pose and preset commands which should be executed on interaction.

**Comment by mrfloris** [comment]
Here's the documentation about CMI IC features
https://www.zrips.net/cmi/commands/interactive-commands/
